src/localize/LocaleMacro.hx

```diff
@@ -9,6 +9,20 @@ class LocaleMacro {
         var fields = Context.getBuildFields();
         for (field in fields) {
             switch (field.kind) {
+                case FVar(type, _):
```

Comment:
I deliberately removed support for vars because I wanted it looks like a function call so people who are using it actually have the idea that it's not just a field access and there's more going on, like catalog lookup. This is to prevent duplicated lookups like: `if (locale.item.length > 5) locale.item.length.substr(0, 5) + "...";` this looks innocent, however it has two catalog lookups and if `item()` was a function, a programmer would want to store it into a var.